open_humans/settings.py

```diff
@@ -416,6 +416,8 @@ def to_bool(env, default="false"):
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
 AWS_STORAGE_BUCKET_NAME = os.getenv("AWS_S3_STORAGE_BUCKET_NAME")
 AWS_DEFAULT_ACL = None  # This will become default in django-storages 2.0
+LOG_BUCKET = os.getenv("LOG_BUCKET")
+AWS_LOG_KEY_BLACKLIST = ["favicon.ico"]
```

Comment:
Why is this in settings.py?

Reply:
Which one?  LOG_BUCKET?  AWS_LOG_KEY_BLACKLIST?  Both?

Reply:
The latter -- if it's not defined by an environment variable, and only used once in the code, I think it's clearer to define something like that it in the file that's using it (e.g. at the top)?

(I don't know if we've had a consistent approach on this, but this was my thinking when seeing it here.)

Reply:
Roger that, I'll move it to the module.